packages/fiori/src/themes/Timeline.css

```diff
@@ -21,3 +21,20 @@
 .ui5-timeline-list-item:last-child {
 	margin-bottom: 0;
 }
+
+:host([axis-orientation="Horizontal"]) .ui5-timeline-list {
+	white-space: nowrap;
+	list-style: none;
+	margin: 0;
+	padding: 0;
+}
+
+:host([axis-orientation="Horizontal"]) .ui5-timeline-list-item {
+	display: inline-block;
+	margin-left: var(--_ui5_tl_li_margin_bottom);
```

Comment:
In RTL you might need to mirror this margin-left with margin-right?

Reply:
Yes, done

Reply:
Done